src/audio.py

```diff
@@ -0,0 +1,53 @@
+import pygame
+import os
+import platform
+
+# Windows misses the dll directory for pygame
+# The below patch fixes it
+if platform.system() == 'Windows':
+    os.add_dll_directory(
+        os.path.dirname(pygame.__file__)
+    )
+
+MASTER = pygame.mixer
+MASTER.init()
+
+
+# This class is intended to be used as a controller
+# that controls a universal mixer.
+# The interface allows devs to add sounds as objects
+# with some control over volume without blocking other sounds.
+class TwistedSound:
+    machine_gun1 = "assets/audio/machine_gun1.ogg"
+
+    # channel.play
+    # channel.stop
+    # channel.pause
+    # channel.unpause
+    @property
+    def channel(self):
+        return self._channel
+
+    @property
+    def volume(self):
+        return self._volume
+
+    # A value of 1 is very loud
+    @volume.setter
+    def volume(self, value):
```

Comment:
`def volume()` has been removed. I didn't understand the colon. I think I'm starting to.

Reply:
Yeah, the colon is a way to post-fix a type annotation.  It can be used on function/method parameters, fields/attributes, and local variables.

Generally function parameters should always have them unless static analysis has another way to know the type.  For example, `self` never needs an annotation cuz the type is always that of the containing class.